### PR TITLE
links updates

### PR DESCRIPTION
**Pull request by Neustradamus**
Update to https and update to good links

### CODE REVIEW

**Comment by Neustradamus** [comment]
@Ri0n: Updated.

**Comment by tehnick** [comment]
@Ri0n Use "Squash and merge" button for megring of this PR please.

File: src/mainwin.cpp [hunk 1]
```diff
@@ -1033,7 +1033,7 @@ void MainWin::actOnlineWikiActivated ()
 
 void MainWin::actOnlineHomeActivated ()
 {
-    DesktopUtil::openUrl("https://psi-im.org");
+    DesktopUtil::openUrl("https://psi-im.org/");
```
Comment (Ri0n) [inline src/mainwin.cpp]: I asked to rollback the change. and similar below

File: qa/integration/psi-data/profiles/default/accounts.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE accounts>
-<accounts version="0.15-dev (Feb  3 2010)" xmlns="http://psi-im.org/options">
+<accounts version="" xmlns="http://psi-im.org/options">
```
[the remark this answers is not in the document]
Comment (Neustradamus) [reply qa/integration/psi-data/profiles/default/accounts.xml]: Version is auto-updated by actual version

File: iconsets/activities/default/icondef.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@
                 <name>Psi+ Activities - size 16</name>
                 <version>0.3</version>
                 <creation>2010-07-18</creation>
-                <home>http://psi-dev.googlecode.com/</home>
+                <home>https://psi-im.org/</home>
```
Comment (tehnick) [inline iconsets/activities/default/icondef.xml]: Why did you change Psi+ URL to Psi IM URL?

File: iconsets/affiliations/default/icondef.xml [hunk 4]
```diff
@@ -5,7 +5,7 @@
 		<version>0.1</version>
 		<description>Adium Affiliations</description>
 		<creation>2010-06-10</creation>
-		<home>http://code.google.com/p/psi-dev</home>
+		<home>https://psi-im.org/</home>
```
Comment (tehnick) [inline iconsets/affiliations/default/icondef.xml]: Why did you change Psi+ URL to Psi IM URL?

File: iconsets/activities/default/icondef.xml (hunk 3 above)
Comment (Neustradamus) [reply iconsets/activities/default/icondef.xml]: Now it is in Psi code, you can look in README: https://github.com/psi-im/psi/blob/master/README.md

File: iconsets/affiliations/default/icondef.xml (hunk 4 above)
Comment (Neustradamus) [reply iconsets/affiliations/default/icondef.xml]: Now it is in Psi code, you can look in README: https://github.com/psi-im/psi/blob/master/README.md

Comment (tehnick) [reply iconsets/affiliations/default/icondef.xml]: They were created in Psi+ project and came from it. I think this link should point to Psi+ project for documenting of this fact.

@Ri0n What is your opinion of this?

File: iconsets/activities/default/icondef.xml (hunk 3 above)
Comment (tehnick) [reply iconsets/activities/default/icondef.xml]: See: https://github.com/psi-im/psi/pull/417#discussion_r261973263

File: iconsets/affiliations/default/icondef.xml (hunk 4 above)
Comment (Ri0n) [reply iconsets/affiliations/default/icondef.xml]: The question if we want to keep origin while the icons are maintained under Psi IM now.
From a user perspective to have a link to the resource where the icons are maintained would be better.

**Comment by Neustradamus** [comment]
Any news?
